docs/semgrep-cloud-platform/teams.md

```diff
@@ -0,0 +1,181 @@
+| API                     | ❌ No     | ❌ No      | ✅ Yes   |       |
+
+## How team access works
+
+- Members of a top-level team gain access to the projects of its subteams. They are indirect members of a subteam.
+- Members of a subteam do not have access to the projects of teams or subteams above it.
+
+In the following diagram, team 1 gains access to subteam 1b's projects, but team 1b does not gain access to projects from team 1.
+
+
+![Team scopes diagram](/img/access-diagram.png#md-width)
+
+- The members Alexis, Pam, and Raj have access to the following projects:
+    - App
+    - Microservices
+    - Frontend
+- The members David, Sebas, and Phaedra have access to the following projects:
+    - Frontend
+
+### The manager role
+
+Use the **manager role** to delegate the assignment of projects across many users. Managers can speed up the deployment of Semgrep into your organization by creating subteams to grant members access to projects.
+
+Given a security engineer who is a manager for **team A** but member for **team B**, with both teams having the same projects:
+
+- The security engineer has manager **access** to the projects.
+- The security engineer can't create subteams for team B but can create subteams for team A.
+
+Additionally, the manager role is able to perform the following:
+
+- Scan new projects through the **Projects** page.
+- Edit projects that their team is assigned to.
+
+Managers cannot remove themselves from their team. Only admins can remove managers.
+
+![A manager's view of the Projects page.](/img/access-manager-view.png)
+**Figure**. A manager's view of the Projects page. They are able to scan new projects and edit the settings for Projects assigned to Teams they are managers of.
+
+## Tips for creating teams and subteams
+
+:::info
+The Semgrep team recommends that admins assign projects to one team only.
+:::
+
+- **Use subteams to grant access to a specific department's repositories**: Create a top-level team for managers or security engineers in your organization who have broad access to a variety of repositories, then create subteams for members to grant them limited access to their specific department's repositories.
+- **Use flat teams to grant access to central projects that are used by a broad group of developers**: It is best to create a separate flat team, without any subteams, and grant the users access to foundational or central repositories from that team. For example, projects that all engineers commit to can be named the Engineering Team.
+
+
+## Configure your teams
+
+### View your teams
+
+You must be an admin or manager to view the **Teams** tab.
+
+1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep Cloud Platform](https://semgrep.dev/login).
+1. Click **[<i class="fa-solid fa-gear"></i> Settings > Access > Teams](https://semgrep.dev/orgs/-/settings/access/teams)**.
+
+### Create a team
+
+1. Click **New team**. The **Create New Team** form appears.
```

Comment:
(This applies to all of the instructions sections) Does it make sense to include the log in and the Settings > Access > Teams instructions at the top of each list just in case someone lands directly at that section and not at the H2?

Reply:
We could do something like the [Policies page](http://semgrep.dev/docs/semgrep-code/policies/), in which we say "In the Settings > Access > Teams tab" and provide a link to the tab. I'll add that in.